06-statistical-testing.Rmd

```diff
@@ -427,8 +429,8 @@ chi_ex1 <- anes_des_educ %>%
 chi_ex1
 ```
 
-The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; $p-value=$ `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:
-<!--This doesn't render correctly in HTML but does ok in PDF.-->
+The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ( $\chi^2=$ `r chi_ex1$statistic`; $p-value=$ `r signif(chi_ex1$p.value,3)` ). To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:  
+<!--This doesn't render correctly in HTML but does ok in PDF.-->  
```

Comment:
Not sure what you don't think looks good in HTML. Share details/screenshot in Google doc or email